### PR TITLE
Completes 94365574-Rake task to copy default views

### PR DESCRIPTION
**Pull request by hpjaj**
@benlangfeld : ready for your review

### CODE REVIEW

**Comment by hpjaj** [comment]
FYI, as I stated in the `USAGE` file, to run the generator, cd into `talking-stick-demo` and run `rails generate default_views`

File: lib/generators/default_views/USAGE [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Description:
```
Comment (bklang) [inline lib/generators/default_views/USAGE]: Is this USAGE file special? Or just the place you chose to keep these notes?

I'd definitely suggest adding this to the README at the top of the repository.

**Comment by hpjaj** [comment]
@bklang  It is a default file included as part of Rails' generator package.  The documentation recommends using it for usage instructions.  That said, agreed, it should also go in the README.  I'll add there, too.

**Comment by bklang** [comment]
:+1: Thanks

**Comment by hpjaj** [comment]
@bklang Yep. I'll update.

File: lib/generators/talking_stick/views/views_generator.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class TalkingStick::ViewsGenerator < Rails::Generators::Base
+  source_root File.expand_path('../templates', __FILE__)
+
+  def generate_participant_views
+    copy_file "#{copy_path}/participants/_form.html.erb", "#{paste_path}/participants/_form.htlm.erb"
+    copy_file "#{copy_path}/participants/edit.html.erb", "#{paste_path}/participants/edit.htlm.erb"
+    copy_file "#{copy_path}/participants/index.html.erb", "#{paste_path}/participants/index.htlm.erb"
+    copy_file "#{copy_path}/participants/new.html.erb", "#{paste_path}/participants/new.htlm.erb"
+    copy_file "#{copy_path}/participants/show.html.erb", "#{paste_path}/participants/show.htlm.erb"
```
Comment (benlangfeld) [inline lib/generators/talking_stick/views/views_generator.rb]: `htlm`?

Comment (hpjaj) [inline lib/generators/talking_stick/views/views_generator.rb]: What, you haven't heard of it?

File: lib/generators/talking_stick/views/USAGE [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Run:
+  rails generate talking_stick:views
+
+This will create the following default views:
+  app/views/talking_stick/participants/_form.htlm.erb
+  app/views/talking_stick/participants/edit.htlm.erb
+  app/views/talking_stick/participants/index.htlm.erb
+  app/views/talking_stick/participants/new.htlm.erb
+  app/views/talking_stick/participants/show.htlm.erb
```
Comment (benlangfeld) [inline lib/generators/talking_stick/views/USAGE]: Here too ;)

Comment (hpjaj) [inline lib/generators/talking_stick/views/USAGE]: Cr*p!  Copy and pasted.  My bad.  Fixing...

File: lib/generators/talking_stick/views/views_generator.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  def generate_participant_views
+    copy_file "#{copy_path}/participants/_form.html.erb", "#{paste_path}/participants/_form.html.erb"
+    copy_file "#{copy_path}/participants/edit.html.erb", "#{paste_path}/participants/edit.html.erb"
+    copy_file "#{copy_path}/participants/index.html.erb", "#{paste_path}/participants/index.html.erb"
+    copy_file "#{copy_path}/participants/new.html.erb", "#{paste_path}/participants/new.html.erb"
+    copy_file "#{copy_path}/participants/show.html.erb", "#{paste_path}/participants/show.html.erb"
+  end
+
+  def generate_room_views
+    copy_file "#{copy_path}/rooms/_form.html.erb", "#{paste_path}/rooms/_form.html.erb"
```
Comment (bklang) [inline lib/generators/talking_stick/views/views_generator.rb]: Do we have to enumerate every single file? Or does it permit globs?

Comment (benlangfeld) [inline lib/generators/talking_stick/views/views_generator.rb]: @hpjaj If you could use globs for this, we can get it merged :)

Comment (hpjaj) [inline lib/generators/talking_stick/views/views_generator.rb]: @benlangfeld well, I can, but I need to figure out how to do it.  do you want me to switch gears and start working on this?

Comment (benlangfeld) [inline lib/generators/talking_stick/views/views_generator.rb]: Just whenever you get a chance :)

File: lib/generators/talking_stick/views/views_generator.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
+class TalkingStick::ViewsGenerator < Rails::Generators::Base
+  source_root File.expand_path('../templates', __FILE__)
```
Comment (benlangfeld) [inline lib/generators/talking_stick/views/views_generator.rb]: What if you set the `source_root` to `copy_path`, something like this:

``` ruby
class TalkingStick::ViewsGenerator < Rails::Generators::Base
  source_root File.expand_path('../../../../app/views/talking_stick', __FILE__)

  def generate_participant_views
    directory "participants", "#{paste_path}/participants"
  end

  def generate_room_views
    directory "rooms", "#{paste_path}/rooms"
  end

private

  def paste_path
    'app/views/talking_stick'
  end
end
```

Does that work? I don't think the current `source_root` is valuable.

Comment (hpjaj) [inline lib/generators/talking_stick/views/views_generator.rb]: Good call @benlangfeld .  I'll update and push.

**Pull request merged**